tracer_options.go

```diff
@@ -73,12 +73,9 @@ func (tracerOptions) PoolSpans(poolSpans bool) TracerOption {
 	}
 }
 
-// HostIPv4 creates a TracerOption that identifies the current service/process.
-// If not set, the factory method will obtain the current IP address.
+// Deprecated: We no longer see reason for exposing this option to the user.
 func (tracerOptions) HostIPv4(hostIPv4 uint32) TracerOption {
-	return func(tracer *tracer) {
-		tracer.hostIPv4 = hostIPv4
-	}
+	return func(tracer *tracer) {}
```

Comment:
I think you can leave this function unchanged, because if you cannot get the current IP at least this one will be respected by zipkin converter